pdc.gemspec

```diff
@@ -16,24 +16,22 @@ Gem::Specification.new do |s|
 
   # Prevent pushing this gem to RubyGems.org by setting 'allowed_push_host', or
   # delete this section to allow pushing this gem to any host.
-  if s.respond_to?(:metadata)
-    s.metadata['allowed_push_host'] = 'https://rubygems.org'
-  else
-    raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
-  end
+  raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.' \
+      unless s.respond_to?(:metadata)
 
+  s.metadata['allowed_push_host'] = 'https://rubygems.org'
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_dependency 'activesupport', '~> 3.2.22.2'
```

Comment:
Are these version changes part of the styling changes?

Reply:
Yeah. I think it is styling change. It only request the user to add dependency in alphabet order

Reply:
True. Sorry my mistake

Reply:
np at all. Thanks for the quick response